PcBdsPkg/PcBdsPkg.dec

```diff
@@ -125,6 +125,9 @@
   # @Prompt FFS Name of Shell Application
   gPcBdsPkgTokenSpaceGuid.PcdShellFile|{ 0xB7, 0xD6, 0x7A, 0xC5, 0x15, 0x05, 0xA8, 0x40, 0x9D, 0x21, 0x55, 0x16, 0x52, 0x85, 0x4E, 0x37 }|VOID*|0x40000129
 
+  ## GUID of the FV to locate the shell in.

```

Comment:
I think it would be helpful to mention that this is optional here. Prevents having to look up the PCD logic in the code to figure out how it is used.

Reply:
Updated.